templates/zerver/api/writing-bots.md

```diff
@@ -430,8 +432,6 @@ Once you have written a test suite, you want to verify that everything works as
   https://github.com/zulip/python-zulip-api/tree/master/zulip_bots/zulip_bots/bots):
   `tools/test-bots <botname>`
 
-* To run any test: `python -m unittest -v <package.bot_test>`
```

Comment:
This can be done with `tools/test-bots <bot-name>`.

Reply:
yep, and see the line 431

Reply:
Ah right, missed that.